packages/oc-template-handlebars-compiler/lib/compile.js

```diff
@@ -20,6 +20,7 @@ const path = require('path');
 // verbose,
 // watch,
 module.exports = (options, callback) => {
+  options.componentPackage = _.cloneDeep(options.componentPackage);
```

Comment:
This is for making sure the compilation starts with a clean object. Some tests were conflicting occasionally as the input object was shared between tests, and at the end of each compilation the object would be re-worked in different ways.